doc/user-manual/language/coinduction.lagda.rst

```diff
@@ -43,9 +43,6 @@ elements of some type ``A`` as follows:
 As opposed to :ref:`inductive record types <recursive-records>`, we have to introduce the keyword
 ``coinductive`` before defining the fields that constitute the record.
 
-It is interesting to note that it is not necessary to give an explicit
```

Comment:
Why has this sentence been deleted?

Reply:
Because it was replaced by more extensive documentation.

I've now put it back (with a cross-reference).